src/core/sagas/search.js

```diff
@@ -0,0 +1,41 @@
+// Disabled because of
+// https://github.com/benmosher/eslint-plugin-import/issues/793
+/* eslint-disable import/order */
+import { call, put, select, takeEvery } from 'redux-saga/effects';
+/* eslint-enable import/order */
+
+import { searchLoad } from 'core/actions/search';
+import { search as searchApi } from 'core/api';
+import { SEARCH_STARTED } from 'core/constants';
+import log from 'core/logger';
+import { createErrorHandler, getState } from 'core/sagas/utils';
+
+
+export function* fetchSearchResults({ payload }) {
+  const { errorHandlerId } = payload;
+  const errorHandler = createErrorHandler(errorHandlerId);
+
+  yield put(errorHandler.createClearingAction());
+
+  try {
+    const { filters, page } = payload;
+
+    const state = yield select(getState);
+
+    const response = yield call(searchApi, {
+      api: state.api,
+      auth: state.auth,
+      filters,
+      page,
+    });
+
+    yield put(searchLoad({ page, filters, ...response }));
+  } catch (error) {
+    log.warn(`Search results failed to load: ${error}`);
+    yield put(errorHandler.createErrorAction(error));
+  }
+}
+
+export default function* searchSaga() {
+  yield takeEvery(SEARCH_STARTED, fetchSearchResults);
```

Comment:
Does this need to process parallel search requests? I don't think we have any page that does that currently. Maybe `takeLatest` is enough so that the last request gets cancelled?